samples/LoggingTracer/LoggingTracer/Logger.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace LoggingTracer
+{
+    using System;
+    using System.Threading;
+
+    public static class Logger
+    {
+        private static DateTime startTime = DateTime.UtcNow;
+
+        static Logger() {
```

Comment:
`{` on new line

did you enable stylecop for the project? (yes, i hate it too!)

Reply:
yes, just overlooked this one.

enforcing coding style is ok, but the "Element should be documented" rule drives me crazy. I don't think I should apply that rule for the sample project. It is ok for public OpenTelemetry API, but not for all implementation IMHO.

Reply:
I'm currently working on cleaning up the codebase in #131, If it's ok I'd like to remove this rule too and set more emphasis on good naming!

Reply:
@lmolkova @SergeyKanzhelev would you be fine with removing the "Element should be documented" rule?